.github/workflows/external.yml

```diff
@@ -22,9 +22,10 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Build and push MLflow Docker image
-        uses: docker/build-push-action@v5
+        uses: docker/build-push-action@v6
         with:
           context: mlflow
+          push: true
```

Comment:
This is now needed for newer versions of the `docker/build-push-action` (version 5 also requires this but since we haven't been pushing new mlflow images for years no one noticed this) since the default value is `false` which causes the image to be built but not pushed. See https://github.com/docker/build-push-action?tab=readme-ov-file#customizing for more details.